README.md

```diff
@@ -32,19 +32,18 @@ To get a general glance of what is included and how it works, follow [this guide
 
 ## Archipelago 101 (and up)
 Now that you have Archipelago running, the following guides will help you master the basics.
-
 * [Ingesting Your First Object](docs/firstobject.md)
-* [Webforms In Archipelago](webforms.md)
-* [Creating Display Modes](docs/createdisplaymodes.md)
-* [_(*Work-In-Progress Page)_ Managing Metadata Displays with Twig](metadatatwigs.md)
 * [General Q&A](docs/generalqa.md)
+* [Webforms In Archipelago _(*Work-In-Progress Documentation)_](docs/webforms.md)
+* [Creating Display Modes](docs/createdisplaymodes.md)
+* [Managing Metadata Displays with Twig _(*Work-In-Progress Documentation)_](docs/metadatatwigs.md)
 * [Debugging PHP in Archipelago](docs/xdebug.md)
 * [Archipelago's File Persistence Strategy](docs/archifilepersistencestrategy.md)
-* [_(*Work-In-Progress Page)_ How to Setup SSL for Docker/Archipelago](sslsetup.md)
+* [How to Setup SSL for Docker/Archipelago _(*Work-In-Progress Documentation)_](docs/sslsetup.md)
 
 ## Archipelagos in the Wild
 Explore Archipelago instances running free across digital realms.
-* [Archipelago Community](inthewild.md)
+* [Archipelago Community Showcase](docs/inthewild.md)
 * [METRO + Archipelago](http://archipelago.nyc)
 
 # Contributing
```

Comment:
Can you please add your name to "Caring & Coding + Fixing" and also change this to "Caring, Documenting, Coding + Fixing". Documenting is in pair to coding to deservers its own mention. Thanks!

Reply:
Ok, will do, thank you.